src/services/apiTypes.ts

```diff
@@ -111,6 +111,25 @@ export interface ChecklistResponse {
     customCheckItems: CustomCheckItem[];
 }
 
+export interface ChecklistPreview {
```

Comment:
hvorfor brukes interface her og ikke type som er brukes på resten?
må si jeg er liker personlig bedre interfaces for objekter og typer mer for enklere ting som f.eks

`type VoidFunction = ()=> void;`

Reply:
Vi har brukt interface for return typer fra APIet og type for component props. Men hvorfor det er blitt sånn er jeg ikke sikker på. Det var jeg som la inn prop types, men kan ikke huske helt hvorfor jeg gikk for type over interface. I ny front end brukes både type og interface for props, kan være det var type jeg hadde brukt for det sist?